### PR TITLE
Remove required terraform-cloud specific variables

### PR DESCRIPTION
**Pull request by AlexDragonDrop**

### CODE REVIEW

**Review by GoodmanBen — COMMENTED** [review]
If they specify that Terraform Clouds is the backend, can you please add a check during job initialization that enforces that in that case the organization name and token are present?

File: main/job_config.go [hunk 1]
```diff
@@ -85,6 +88,19 @@ type JobConfig struct {
 	ResourcesBlackList terraformValueObjects.ResourceNameList
 }
 
+// validateJobConfig validates the JobConfig struct with the values as expected.
+func validateJobConfig(config JobConfig) error {
+	if config.StateBackend == "TerraformCloud" {
```
Comment (GoodmanBen) [inline main/job_config.go]: Let's make this a bit more to just by lower casing config.stateBackend and comparing against "terraformcloud"

**Review by GoodmanBen — COMMENTED** [review]
One comment, otherwise looks good

File: main/job_config.go (hunk 1 above)
Comment (AlexDragonDrop) [reply main/job_config.go]: good point

**Review by GoodmanBen — APPROVED** [review]

**Pull request merged**